private/buf/bufctl/controller.go

```diff
@@ -129,6 +130,7 @@ type Controller interface {
 		defaultMessageEncoding buffetch.MessageEncoding,
 		options ...FunctionOption,
 	) error
+	GetWKTBucket(ctx context.Context) (storage.ReadBucket, error)
```

Comment:
This shouldn't be part of the `Controller` really - instead, expose `bufcli.newWKTStore` as public `bufcli.NewWKTStore` (ie just `s/n/N/`) and call this function in `buflsp` to make a `bufwktstore.Store`.

Reply:
Done